Source/WebCore/platform/mediastream/gstreamer/GStreamerRTPPacketizer.cpp

```diff
@@ -177,7 +177,7 @@ String GStreamerRTPPacketizer::rtpStreamId() const
         return emptyString();
 
     if (auto rid = gstStructureGetString(m_encodingParameters.get(), "rid"_s))
-        return rid.toString();
+        return String(rid.span());
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?